src/main/java/commands/Parser.java

```diff
@@ -0,0 +1,51 @@
+package commands;
+
+public class Parser {
+    public static Command parse(String input) {
+        String[] options = input.split(" /");
+
+        String[] mainPart = options[0].split(" ", 2);
```

Comment:
Perhaps you can change this variable name to be a plural form of a different variable name

Reply:
Actually I intended it to be the main part with several sections within it, so I considered plural form not necessary. On second thought now `mainPart` is not a clear variable name in the first place. Renamed.